canon/CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+# 0.6.0
+
+## Changed
+
+Deprecate the usage of the `Store` trait.
+
+the trait `Canon<S>` becomes `Canon`.
+
+`Canon::<S>::write(&type, &mut Sink<S>)?;` becomes `type.encode(&mut sink);`
+
+Writes are assumed to always succeed, analgous to how a push to a `Vec` in rust can only panic with out of memory exceptions, but is assumed to be failsafe.
+
+The names encode/decode is choosen not to confuse them with `std::io::{Read, Write}`.
+
+# 0.5.3 a9b53e4e0ebe96fa7bd90ce2ce03875d5e21124a
```

Comment:
Since you added the tag, you don't need the commit's sha close to the version, but just make the version a like such as:
```
## [0.5.3] - 2020-10-10
```

And then add the link in MD at the end of the file (see again [changelog format](https://keepachangelog.com/en/1.0.0/) for further detail and / or  [dusk-abi's changelog](https://github.com/dusk-network/dusk-abi/blob/master/CHANGELOG.md) as example).

Also, the date is missing in all the versions.